Sources/BraintreeShopperInsights/BTShopperInsightsClient.swift

```diff
@@ -27,16 +27,17 @@ public class BTShopperInsightsClient {
     
     /// This method confirms if the customer is a user of PayPal services using their email and phone number.
     /// - Parameters:
-    ///   - request: A `BTShopperInsightsRequest` containing the buyer's user information
+    ///   - request: Required:  A `BTShopperInsightsRequest` containing the buyer's user information
+    ///   - experiment: Optional:  A `JSONObject` passed in as a string indicating details of the experiment
     /// - Returns: A `BTShopperInsightsResult` instance
     /// - Warning: This feature is in beta. Its public API may change or be removed in future releases.
     ///         PayPal recommendation is only available for US, AU, FR, DE, ITA, NED, ESP, Switzerland and UK merchants.
     ///         Venmo recommendation is only available for US merchants.
-    public func getRecommendedPaymentMethods(request: BTShopperInsightsRequest) async throws -> BTShopperInsightsResult {
+    public func getRecommendedPaymentMethods(request: BTShopperInsightsRequest, experiment: String? = nil) async throws -> BTShopperInsightsResult {
```

Comment:
What do you think about creating a local property for this so merchants don't need to resend the experiment in the presented events? We can just pass the initial experiment down as needed and presented can just take in the button rank?

Reply:
yeah i like that idea! feel like it reads more clearly too - i've updated iit here >> 6a9b7c21a137b8e490f4ca0e1fc3e0917ec92624